core/src/history/events.rs

```diff
@@ -84,7 +88,7 @@ impl EventRegistry {
 
     pub fn write_to_file(&self, path: impl AsRef<Path>) -> Result<()> {
         // Write to tmp file until complete and then rename.
-        let temp_path = path.as_ref().with_extension(".temp");
+        let temp_path = path.as_ref().with_extension("temp");
```

Comment:
Why this change?

Reply:
I noticed while testing that we were creating a `orderbookfile..temp` instead of `orderbookfile.temp`. I just threw in the fix here since it is pretty insignificant but can split it out if you think its worth it.

Reply:
no that's fine, was just unexpected, so I wanted to understand why.